### PR TITLE
Add exception to AddAction when the map is part of an asset

### PR DESCRIPTION
**Pull request by jimon**
Show a better exception what trying to add an action to the map which is part of an input asset which other maps being enabled. Not sure if we need a changelog entry here.

### CODE REVIEW

File: Assets/Tests/InputSystem/CoreTests_Editor.cs [hunk 1]
```diff
@@ -676,6 +676,41 @@ public void Editor_InputAsset_CanReplaceBindingGroupThroughSerialization()
         Assert.That(map.bindings[0].groups, Is.EqualTo(""));
     }
 
+    [Test]
+    [Category("Editor")]
+    public void Editor_InputAsset_AddingActionToAssetWithMultipleMaps()
```
Comment (Rene-Damm) [inline Assets/Tests/InputSystem/CoreTests_Editor.cs]: This behavior is not specific to the editor. Should be in `CoreTests_Actions`.

Also, the test name is just a name and doesn't describe behavior. Something `Actions_AddingActionToAssetWithEnabledActions_ThrowsException`.

**Review by Rene-Damm — CHANGES_REQUESTED** [review]
Needs CHANGELOG entry. Also needs updating of the xmldoc comment.

File: Assets/Tests/InputSystem/CoreTests_Editor.cs (hunk 1 above)
Comment (jimon) [reply Assets/Tests/InputSystem/CoreTests_Editor.cs]: Ah sorry, missed the fact that CoreTests_Actions had InputActionAsset related tests. Fixed.

**Comment by jimon** [comment]
Added CHANGELOG entry and updated xmldoc comment

**Pull request merged**